include/swift/AST/DiagnosticsSema.def

```diff
@@ -7032,6 +7032,10 @@ WARNING(variable_never_mutated, none,
         "variable %0 was never mutated; "
         "consider %select{removing 'var' to make it|changing to 'let'}1 constant",
         (Identifier, bool))
+WARNING(variable_tuple_elt_never_mutated, none,
+        "variable %0 was never mutated in tuple; "
+        "consider changing pattern to 'case (..., let %1, ...)'",
```

Comment:
The first part here talks about tuple and the other about pattern. let's remove `in tuple` and use `the pattern` instead of just `pattern`.

Reply:
Hi @xedin , I have made the changes and pushed it.